web/src/components/dashboard/SidebarCardsList.tsx

```diff
@@ -65,10 +67,15 @@ export const SidebarCardsList = (props: SidebarCardsListProps): ReactElement =>
           <hr className="margin-top-3 bg-cool-lighter" aria-hidden={true} />
           <div className="desktop:margin-bottom-1">
             <Accordion
+              ref={accordionRef}
               expanded={hiddenAccordionIsOpen}
               onChange={(): void => {
-                if (!hiddenAccordionIsOpen) {
+                if (hiddenAccordionIsOpen) {
+                  window.scroll(0, accordionPosition);
```

Comment:
Was this behavior discussed with the product/design team? It feels a little jarring to have the page jump when I close the accordion.

If this was just the interpretation of the comment from myself and JP - I think the intention is just that _no_ scroll behavior should happen on close. This would also make the code a lot simpler because then you would not need to track the accordionPosition variable at all.

If you did discuss this with product/design and this is desired behavior, then you can ignore this comment.

Reply:
I did this based on the comments from you and JP (and what I saw happening in the app).